src/GovUk.Education.ExploreEducationStatistics.Data.Seed/Services/SeedService.cs

```diff
@@ -31,44 +41,140 @@ public void Seed()
+                                        TopicId = topic.Id,
+                                        Topic = new Topic
+                                        {
+                                           Id = topic.Id,
+                                           Slug = topic.Slug,
+                                           Title = topic.Title,
+                                           ThemeId = theme.Id,
+                                           Theme = new Theme
+                                           {
+                                              Id = theme.Id,
+                                              Slug = theme.Slug,
+                                              Title = theme.Title
+                                           }
+                                        }
+                                    },
+                                    Subjects = new []
+                                    {
+                                        new Subject {
+                                            Id = subject.Id,
+                                            Name = subject.Name,
+                                            ReleaseId = release.Id
+                                        }
+                                    }
+                                };
 
-                    var subjects = updated.Topics
-                        .SelectMany(topic => topic.Publications)
-                        .SelectMany(publication => publication.Releases)
-                        .SelectMany(release => release.Subjects);
+                                var file = SamplePublications.SubjectFiles.GetValueOrDefault(subject.Id);
 
-                    Seed(subjects);
+                                StoreFiles(r, file, subject.Name);
+                                Seed(file.ToString() + ".csv", r);
+                            }   
+                        }
+                    } 
                 }
             }
-            finally
-            {
-                _context.ChangeTracker.AutoDetectChangesEnabled = true;
-            }
 
             stopWatch.Stop();
             _logger.LogInformation("Seeding completed with duration {duration} ", stopWatch.Elapsed.ToString());
         }
 
-        private void Seed(IEnumerable<Subject> subjects)
+        private void Seed(string dataFileName, Release release)
         {
-            foreach (var subject in subjects)
+            var storageAccount = CloudStorageAccount.Parse(_storageConnectionString);
+            var client = storageAccount.CreateCloudQueueClient();
+            var pQueue = client.GetQueueReference("imports-pending");
+            var aQueue = client.GetQueueReference("imports-available");
+            
+            pQueue.CreateIfNotExists();
+            aQueue.CreateIfNotExists();
+            
+            var message = BuildMessage(dataFileName, release);
+            
+            pQueue.AddMessage(message);
```

Comment:
I think there's potentially a problem here @SiShakes because if these messages are processed in a different order to what they are generated in, then the filter and indicator id's are going to be different to what they have previously been observed as. Queries containing these id's have been persisted in Table and Chart configuration embedded in the Content API data (see `SubjectFilterItemIds` and `SubjectIndicatorIds` in ApplicationDbContext of the ContentAPI project. The id's also feature heavily in the Postman API tests.

Reply:
I will change the seeder to try to process sequentially on a new ticket.